tools/bundler/index.js

```diff
@@ -29,14 +31,65 @@ console.log(`Using the following output directory: ${outputDirectory}`);
       const filePathToBundle = `file://${versionDir}/asyncapi.json`;
       const fileToBundle = await Bundler.get(filePathToBundle);
       const bundledSchema = await Bundler.bundle(fileToBundle);
+      modifyRefsandDefinitions(bundledSchema);
```

Comment:
It took me a few seconds to understand what it meant. I was thinking: "what the hell is a Refsand Definition now?" 😂 

```suggestion
      modifyRefsAndDefinitions(bundledSchema);
```